### PR TITLE
Add missing models to README and builder + fixes

### PR DESCRIPTION
**Pull request by hurda**
Added a couple of models to the README and fixed the listing of the Japanese 4.0 models.

Added models to the builder-model-list, fixed the 7500-entry and added the other 2.0-BIOS to the BIOS-list.

BIOS 2.0 19950507 (ps-20a) has the same data at the same spot as 19950510 (ps-20e), so I guess it will work just the same?

### CODE REVIEW

File: builder/builder.cc [hunk 1]
```diff
@@ -87,6 +87,7 @@ static std::map<BIOSKey, ExploitSettings> biosExploitSettings{
     // calls payload from ISR, number of loops per increment, index of directory entry, exploit type.
     {{10, 19940922}, {0x801ffcb0, 0x80204f04, 0x0c0006c1, 0x0c002f92, true, 3}},
     {{11, 19950122}, {0x801ffcc0, 0x80204d6c, 0x10000004, 0x10001c36, true, 3, 0, ExploitType::MemcardISR, -0x70cc}},
+    {{20, 19950507}, {0x801ffcb8, 0x80204ef4, 0x0c001ab0, 0x0c002f92, true, 3, 0, ExploitType::ICacheFlush}},
```
Comment (brad-lin) [inline builder/builder.cc:90]: This is not going to work; you can't just copy settings from one BIOS version to another.

I'll add support for this BIOS in the master branch.

File: README.md [hunk 2]
```diff
@@ -75,15 +75,16 @@ As more reliable or faster versions of the exploit are developed, the images are
 |-------------------|--------|------------------------|---------------|
 | 1.0 (1994-09-22)  | SCPH-1000 | **Yes** | [20210421](images/freepsxboot-unirom-fastload-20210421-bios-1.0.mcd) |
 | 1.1 (1995-01-22)  | SCPH-3000 | **Yes** | [20210421](images/freepsxboot-unirom-fastload-20210421-bios-1.1.mcd) |
+| 2.0 (1995-05-07)  | SCPH-1001 | **Yes; see note below** | [20210421](images/freepsxboot-unirom-fastload-20210421-bios-2.0.mcd) |
```
Comment (brad-lin) [inline README.md:78]: Please update the note below with SCPH-1001, as well as the 3rd bullet point in the "Usage" section.

**Comment by brad-lin** [comment]
I have pushed the support for BIOS 2.0 (1995-05-07) and the corresponding image: freepsxboot-unirom-fastload-20210421-bios-2.0-1995-05-07.mcd

Can you rebase your pull request and adapt it? Thanks

**Comment by hurda** [comment]
What a mess. Clean new here: #45

**Pull request closed without merging**